src/main/java/world/inetum/realdolmen/inetumrealJobs/entities/enums/Role.java

```diff
@@ -0,0 +1,6 @@
+package world.inetum.realdolmen.inetumrealJobs.entities.enums;
+
+public enum Role {
+    ROLE_JOB_SEEKER,
+    ROLE_RECRUITER
```

Comment:
The ROLE_ prefix seems not needed? It's already in a Role class. If you need the grant name (which is with the prefix), then you can add it as parameter of the enum.